packages/web/src/styles.ts

```diff
@@ -94,6 +106,15 @@ export const fonts = StyleSheet.create({
     lineHeight: 24,
     color: colors.dark,
   },
+  h5a: {
+    fontFamily: typeFaces.futura,
+    fontSize: 20,
+    lineHeight: 24,
+    fontWeight: '500',
+    color: colors.dark,
+    textRendering: 'geometricPrecision',
+  },
+  // deprecated replaced with h6
   h5: {
```

Comment:
I didnt want to cause the diff on this PR to be even larger by changing all font.h5 refs to fonts.h6. i think this can be done separately